core/consensus/slots/src/lib.rs

```diff
@@ -115,7 +115,7 @@ pub fn start_slot_worker_thread<B, C, W, SO, SC, T, OnExit>(
 		let _ = runtime.block_on(slot_worker_future);
 	});
 
-	result_recv.recv().expect("Aura start thread result sender dropped")
+	result_recv.recv().expect("Slots start thread result sender dropped")
```

Comment:
I know this is not really part of the diff of this PR, but we shouldn't be panicking in this scenario. Rather returning an error or logging a message.

Reply:
I will fix it in a separate PR.  That said, this seems like a bug, so we should probably call `error!`.